### PR TITLE
Split acceptance test OccContext

### LINKED ISSUE #33668: Refactor command line related tests into multiple files
Currently, there are two files related to `occ` or `commandline` related contexts. But, they are expected to grow. So, it would be better if we could refactor them into multiple files/contexts as we do currently on `api` or `webui` tests.  

At present, `CommandLine.php` has ~600 LOC and `OccContext` clocks at ~1000.

### Ways we could refactor:
1. Based on commands
2. ...

### PR DESCRIPTION
**Pull request by phil-davis**
## Description
1) split ``occ`` ``user`` and ``group`` command tests into a separate ``OccUsersGroupsContext``
2) split ``occ`` ``apps`` command tests into a separate ``OccAppManagementContext``

Note: there is an existing suite called ``AppManagementContext`` but that is specifically for test steps that adjust the ``apps_paths`` setting. It has before and after scenario to manage resetting the value etc. So we do not want to add lots of other test steps to that.

The other test steps are for smaller commands with less test steps for local storage, trashbin, versions, log settings... They can be split if they get big.

## Related Issue
- Fixes #33668 

## Motivation and Context
See issue.

## How Has This Been Tested?
CI

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:
- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:
- [ ] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

File: tests/acceptance/features/bootstrap/OccAppManagementContext.php [hunk 1]
```diff
@@ -0,0 +1,208 @@
+<?php
+/**
+ * ownCloud
+ *
+ * @author Phil Davis <phil@jankaritech.com>
+ * @copyright Copyright (c) 2018, ownCloud GmbH
```
Comment (paurakhsharma) [inline tests/acceptance/features/bootstrap/OccAppManagementContext.php]: s/2018/2019

File: tests/acceptance/features/bootstrap/OccUsersGroupsContext.php [hunk 2]
```diff
@@ -0,0 +1,546 @@
+<?php
+/**
+ * ownCloud
+ *
+ * @author Phil Davis <phil@jankaritech.com>
+ * @copyright Copyright (c) 2018, ownCloud GmbH
```
Comment (paurakhsharma) [inline tests/acceptance/features/bootstrap/OccUsersGroupsContext.php]: s/2018/2019

**Review by paurakhsharma — COMMENTED** [review]
LGTM :+1:

**Review by paurakhsharma — APPROVED** [review]

**Comment by phil-davis** [comment]
Backport ``stable10`` #34134

**Review by individual-it — APPROVED** [review]

**Pull request merged**